tests/Licenses/Entities/Offer/PriceBandTest.php

```diff
@@ -1,8 +1,8 @@
 <?php
 
-namespace ArrowSphere\PublicApiClient\Tests\Licenses\Entities\Offer\PriceBand;
+namespace ArrowSphere\PublicApiClient\Tests\Licenses\Entities;
```

Comment:
wrong namespace, should be
```suggestion
namespace ArrowSphere\PublicApiClient\Tests\Licenses\Entities\Offer;
```

Reply:
done